File: ios/Reguerta/Reguerta/Domain/Devices/RegisteredDevice.swift
import Foundation

struct RegisteredDevice: Equatable, Sendable {
    let deviceId: String
    let platform: String
    let appVersion: String
    let osVersion: String
    let apiLevel: Int?
    let manufacturer: String?
    let model: String?
    let fcmToken: String?
    let firstSeenAtMillis: Int64
    let lastSeenAtMillis: Int64
    let tokenUpdatedAtMillis: Int64?
}

File: android/Reguerta/app/src/main/java/com/reguerta/user/domain/devices/RegisteredDevice.kt
package com.reguerta.user.domain.devices

data class RegisteredDevice(
    val deviceId: String,
    val platform: String,
    val appVersion: String,
    val osVersion: String,
    val apiLevel: Int?,
    val manufacturer: String?,
    val model: String?,
    val fcmToken: String?,
    val firstSeenAtMillis: Long,
    val lastSeenAtMillis: Long,
    val tokenUpdatedAtMillis: Long?,
)
